src/ext/ca/serverca/scasched/scauser.cpp

```diff
@@ -515,7 +515,7 @@ HRESULT ScaUserExecute(
         if (wzDomain && *wzDomain)
         {
             er = ::DsGetDcNameW(NULL, wzDomain, NULL, NULL, NULL, &pDomainControllerInfo);
-            if (HRESULT_FROM_WIN32(er) == RPC_S_SERVER_UNAVAILABLE)
+            if (RPC_S_SERVER_UNAVAILABLE == er)
```

Comment:
At least ERROR_NO_SUCH_DOMAIN and RPC_S_SERVER_UNAVAILABLE. 
MSDN doesn't explictly state all possible error codes